app/src/main/java/com/arya/myshoppingapp/SplashActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.arya.myshoppingapp;
+
+
+import android.os.Bundle;
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Handler;
+
+
+public class SplashActivity extends Activity {
+Handler handler;
```

Comment:
declare instance variables as private when variables are not used outside of class